### DISCUSSION TITLE
My attempt at a rigid body system using DPD and the NVE integrator results in either a drop in temperature but constant energy or an increase in energy depending on rigid body count

### DISCUSSION THREAD

**Discussion by TheDoctorWho169** [glotzerlab/hoomd-blue#1767] [Support]
Hello!

I am attempting to create a rigid body system of rectangular bodies in a homogeneous neutral solvent. The density of the system is 3 and the interaction parameters are all 25 with a gamma of 4.5. I initially place the solvent then the rigid bodies but I give the system an stabilization period to allow the solvent particles to be forced from the rigid bodies. After this stabilization period I start collecting thermodynamic data. This data shows inconsistent energy readings from one set of systems to the next. For systems of one rigid body my temperature, which is initially set at 1 kBT, is constant at a decimal lower than 1. For systems of multiple rigid bodies the energy of my system increases until I get a 'particle is out of simulation box' error which I assume is due to the increasing energy "outpacing" my selected timestep. 

I am more than happy to provide more information and snippets of my code to try figure out why this is happening and any potential solutions.

**Comment by joaander** [comment]
HOOMD does not compute the linear velocity resulting from the angular momentum for rigid body constituent particles: #278 Therefore, the pairwise DPD thermostats will not operate correctly with rigid bodies.

**Reply by TheDoctorWho169** [reply]
Thank you!
